docs/mesos/PrepareExternalServices.md

```diff
@@ -0,0 +1,49 @@
+# Prepare External Services for Mesos + Calico Deployment
+
+## Docker
+
+- **Docker must be installed on each Mesos Agent** that is deploying calico via the packaged Calico container (recommended). Users who prefer to [run Calico as a baremetal service](#) do not need to install Docker on each Agent.
+- **Docker must be installed on Mesos Master** if etcd and ZooKeeper are being deployed as Docker containers (as this tutorial does). Users who are running etcd / ZooKeeper elsewhere do not need to install Docker on each Master.
+
+Run the following commands to install Docker:
+
+    $ sudo yum -y install docker docker-selinux
+    $ sudo systemctl enable docker.service
+    $ sudo systemctl start docker.service
+
+### Verify Docker installation
+
+    $ sudo docker run hello-world
+
+*Optional:* You may also want to create a `docker` group and add your local user to the group.  This means you can drop the `sudo` in the `docker ...` commands that follow.
+
+    $ sudo groupadd docker
+    $ sudo usermod -aG docker `whoami`
+    $ sudo systemctl restart docker.service
+
+Then log out (`exit`) and log back in to pick up your new group association.  Verify your user has access to Docker without sudo
+
+    $ docker ps
+
+## ZooKeeper
+Mesos uses ZooKeeper to elect and keep track of the leading master in the cluster.
+
+    $ sudo docker pull jplock/zookeeper:3.4.5
+    $ sudo docker run --detach --name zookeeper -p 2181:2181 jplock/zookeeper:3.4.5
+
+## etcd
+Calico uses etcd as its data store and communication mechanism among Calico components.
+
+etcd needs your fully qualified domain name to start correctly.
+
+    $ sudo docker pull quay.io/coreos/etcd:v2.2.0
+    $ export FQDN=`hostname -f`
+    $ sudo mkdir -p /var/etcd
+    $ sudo docker run --detach --name etcd --net host -v /var/etcd:/data quay.io/coreos/etcd:v2.2.0 \
```

Comment:
Change this to `$ sudo FQDN='hostname-f' docker run --detach ....` (but replace the single quote with a bash backtick)